jobs/github_tag_release.groovy

```diff
@@ -1,38 +0,0 @@
-import util.Plumber
-
-def p = new Plumber(name: 'release/codekit/github-tag-release', dsl: this)
```

Comment:
This one is used in the nightly/weekly/official release jobs.  It can't be removed.

Reply:
Actually, it looks like this job is not used, as the release jobs invoke the same code directly, not via the job.  Removal is OK.